### PR TITLE
Refactor SharedPreferences

### LINKED ISSUE #73: Refactor ArtMakerSharedPreferences
- Refactor ArtMakerSharedPreferences to match the rest of the project's naming system...

### PR DESCRIPTION
**Pull request by emmanuelmuturia**
- Fixes #73 by moving all of the ArtMakerSharedPreferences implementations to PreferencesManager...

### CODE REVIEW

**Comment by Fbada006** [comment]
I do not think this task was about renaming, was it? @CalebKL

**Review by Fbada006 — CHANGES_REQUESTED** [review]
As agreed, this needs more work

**Comment by emmanuelmuturia** [comment]
> As agreed, this needs more work

Let me share my current changes in a moment...

File: artmaker/src/main/java/io/artmaker/ArtMakerViewModel.kt [hunk 1]
```diff
@@ -174,64 +141,46 @@ internal class ArtMakerViewModel(
...
     private fun updateDisableStylusDialog(canShow: Boolean) {
-        preferences.set(PreferenceKeys.PREF_SHOW_DISABLE_STYLUS_DIALOG, canShow)
+        preferencesManager.updateDisableStylusDialog(canShow = canShow)
         _uiState.update {
             it.copy(
-                canShowDisableStylusDialog = preferences.get(
-                    key = PreferenceKeys.PREF_SHOW_DISABLE_STYLUS_DIALOG,
-                    true,
-                ),
+                canShowDisableStylusDialog = canShow,
```
[the remark this answers is not in the document]
Comment (emmanuelmuturia) [reply artmaker/src/main/java/io/artmaker/ArtMakerViewModel.kt:183]: This is noted. I am pushing the latest updates in a moment...

File: artmaker/src/main/java/io/artmaker/ArtMakerViewModel.kt [hunk 2]
```diff
@@ -174,64 +141,46 @@ internal class ArtMakerViewModel(
...
     private fun updateStylusSetting(useStylusOnly: Boolean) {
-        preferences.set(PreferenceKeys.PREF_USE_STYLUS_ONLY, useStylusOnly)
+        preferencesManager.updateStylusOnlySetting(useStylusOnly = useStylusOnly)
         _uiState.update {
             it.copy(
-                shouldUseStylusOnly = preferences.get(
-                    key = PreferenceKeys.PREF_USE_STYLUS_ONLY,
-                    false,
-                ),
+                shouldUseStylusOnly = preferencesManager.getStylusOnlySetting(useStylusOnly = useStylusOnly),
```
Comment (Fbada006) [inline artmaker/src/main/java/io/artmaker/ArtMakerViewModel.kt:156]: The default values of the preferences should not come from the value being used to set the preferences. Use the default values that were with those preferences before. In the case of stylus setting, eg, the default value was false. the getters from the pref manager do not need to have any paraemeters

**Review by Fbada006 — CHANGES_REQUESTED** [review]
Better. Just one more thing to change

**Review by Fbada006 — APPROVED** [review]
Nice work

**Pull request merged**